### DISCUSSION TITLE
Repurposing forward node into a heavy node without full re-install

### DISCUSSION THREAD

**Discussion by presianbg** [Security-Onion-Solutions/securityonion#7213] [Unsupported Versions] [answered]
Hi,

Have anyone tried to "soft" reinstall a forward node into a heavy node ?
Is it possible to transform a `forward` node into `heavy` one by running `sudo SecurityOnion/setup/so-setup iso` ? 

Cheers,
PY

**Comment by dougburks** [comment, marked as answer]
I'm not sure that we've tested this scenario as we typically recommend against heavy nodes for most use cases:
https://docs.securityonion.net/en/2.3/architecture.html#distributed
